cppcache/integration-test-2/RegionPutGetAllTest.cpp

```diff
@@ -198,4 +199,38 @@ TEST(RegionPutGetAllTest, variousPdxTypes) {
   assert_eq(putAllMap, getAllMap);
 }
 
+TEST(RegionPutGetAllTest, nullValue) {
+  Cluster cluster{LocatorCount{1}, ServerCount{2}};
+  cluster.getGfsh()
+      .create()
+      .region()
+      .withName("region")
+      .withType("REPLICATE")
+      .execute();
+
+  auto cache = cluster.createCache();
+  auto region = setupRegion(cache);
+
+  setupPdxTypes(cache);
+
+  HashMapOfCacheable map;
+
+  // Add a null value
+  map.emplace(CacheableInt32::create(PdxTypesHelper1::index),
+              std::shared_ptr<PdxTypesHelper1::type>());
+
+  auto keys = to_keys(map);
+
+  try {
+    region->putAll(map);
+  } catch (CacheServerException) {
```

Comment:
should be able to use `ASSERT_THROW`, no?

Reply:
Good catch. Replaced with EXPECT_THROW.

Reply:
You should be using Expect exception.

Reply:
Done.